projects/control-service/projects/pipelines_control_service/src/main/resources/application.properties

```diff
@@ -149,3 +149,6 @@ datajobs.security.kerberos.kerberosPrincipal=""
 datajobs.security.kerberos.keytabFileLocation=
 datajobs.security.kerberos.krb5ConfigLocation=
 datajobs.security.kerberos.enabled=false
+
+#Settings for OAuth2 authorization provider
+datajobs.security.oauth2.enabled=true
```

Comment:
is this a new property? don't we have already configurable oauth2? could you link to existing oauth2 configurations?

Reply:
@tozka requested to have 2 separate flags for kerberos and oauth2. This commit introduces the flag specifically for oAuth2.

Reply:
I see, is it oauth2 authentication?
Could you link to existing oauth2 configurations?

Reply:
https://github.com/vmware/versatile-data-kit/blob/830e92432313a7739559659424cc24fd81a0f868/projects/control-service/projects/pipelines_control_service/src/main/resources/application-dev.properties#L38-L40